doc/examples.md

```diff
@@ -263,4 +263,30 @@ Setting the hostname of a system is as simple as writing `/etc/hostname`:
 }
 ```
 
+## Adding users
+
+Users can be added to a CoreOS system with the `passwd.users` key which takes a list of objects that specify a given user. If you wanted to configure a user "core" and a user "jenkins" you would do that as follows:
```

Comment:
Can you update this note as well and mention that systemUser must already be created by the OS (Ignition is only updating this user) and that it's also _creating_ the jenkins user?